db/migrate/20230118195424_add_user_ref_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: {to_table: 'users'}
```

Comment:
- [ ] Like the previous reviewer suggested, please ensure that the reference is implemented with the `author_id` as shown in the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)